Filler SQL data

bpowell opened the pull request:
@jelevoir if you want to add more data, be my guest. We can merge this tomorrow when we meet before class. That should give us enough time to add things we missed

bpowell committed:
Filler SQL data

bpowell committed:
All the things are tree fifty

jelevoir commented:
This is a nice start to the filler data. 
I'll add some data tomorrow morning as I get the droplet ready for demonstration.

Commit:
add 3 new bars

bpowell committed:
Changed BWW to Buffalo Wild Wings

bpowell committed:
Made bar addresses real

bpowell committed:
Added more drinks

bpowell commented:
@jelevoir I added more drinks, removed one of the bars, and added proper address/number for the bars. I also added some drinks to Penny Black. If you want to add more go ahead, but I think at this point we good.

bpowell commented:
Merge when ready I guess
